sdk/core/azure-core/src/main/java/com/azure/core/implementation/util/FileContent.java

```diff
@@ -38,24 +39,58 @@
      *
      * @param file The {@link Path} content.
      * @param chunkSize The requested size for each read of the path.
+     * @param position Position, or offset, within the path where reading begins.
+     * @param size Total number of bytes to be read from the path.
      * @throws NullPointerException if {@code file} is null.
-     * @throws IllegalArgumentException if {@code chunkSize} is less than or equal to zero.
+     * @throws IllegalArgumentException If {@code chunkSize} is less than or equal to 0 or {@code position} or
+     * {@code size} is non-null and negative.
      */
-    public FileContent(Path file, int chunkSize) {
-        Objects.requireNonNull(file, "'file' cannot be null.");
+    public FileContent(Path file, int chunkSize, Long position, Long size) {
```

Comment:
Should we consider having a convenience ctor without position and size(what we had before)?